core/app/models/spree/promotion.rb

```diff
@@ -64,9 +64,24 @@ def as_json(options = {})
       super
     end
 
+    def unstarted?
```

Comment:
I may be splitting hairs here, but I think it may be worth being more consistent with naming in order to honor the principle of least surprise :neckbeard: 

We have `not_expired` but also`unstarted`: I think we may prefer to change one of them in order to have the same prefix. So it would be either `unexpired` or `not_started`. What do you think?

Reply:
I will go with `not_started`. what do you think should I update too on the front end?

![image](https://user-images.githubusercontent.com/2718753/55502263-6b83b980-5609-11e9-996c-d3cc8ced6f36.png)

Reply:
👍 yes, I think that it's best to be consistent also there with the chosen naming.